src/js/components/MaskedInput/__tests__/MaskedInput-test.js

```diff
@@ -191,6 +191,55 @@ describe('MaskedInput', () => {
     expect(onChange).toHaveReturnedWith('aa!');
   });
 
+  test('should not enable to type beyond options via keyboard', async () => {
+    const onChange = jest.fn((event) => event.target.value);
+    const { getByTestId, container } = render(
+      <MaskedInput
+        data-testid="test-input"
+        id="item"
+        name="item"
+        mask={[
+          {
+            options: ['aaa', 'aba', 'abb'],
+            regexp: /\w$/,
+          },
+        ]}
+        onChange={onChange}
+      />,
+    );
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
```suggestion
```
I don't think we need a snapshot here since it isn't capturing any behavior we are trying to specifically test for in this test